app/controllers/posts_controller.rb

```diff
@@ -8,5 +8,31 @@ def index
   def show
     @user = User.find(params[:user_id])
     @post = @user.posts.find(params[:id])
```

Comment:
- [ ] Please make sure to create a method called `current_user` in your ApplicationController as requested in the project requirements. (You're using this method in other controllers while create actions but it's not defined).
  - You can consider using the code.
```ruby
def current_user
  User.first
end
```
  
![image](https://github.com/yayaismaail/blog/assets/83803414/cc6d3751-f2df-4497-91ed-5d89de5ae917)